pkg/vpcmodel/output.go

```diff
@@ -64,7 +67,16 @@ func NewOutputGenerator(c *VPCConfig, grouping bool, uc OutputUseCase, archOnly
 	return res, nil
 }
 
-func (o *OutputGenerator) Generate(f OutFormat, outFile string) (string, error) {
+// VPCAnalysisOutput captures output per VPC
+type VPCAnalysisOutput struct {
+	VPCName    string
+	Output     string
+	jsonStruct interface{}
+	format     OutFormat
+}
+
+// Generate returns VPCAnalysisOutput for its VPC analysis results
+func (o *OutputGenerator) Generate(f OutFormat, outFile string) (*VPCAnalysisOutput, error) {
```

Comment:
Is there a reason for `Generate` to still get an `outFile`?
If it's only for testing, I would find other solutions.
Currently, formatters call `writeOutput` or `WriteToFile` (do we really need both?) without any effect.

Reply:
- Deleted function `writeOutput`
- Actually `Generate` should still get an `outFile` because of the `drawio` format -- the aggregation does not yet support this format.. added a fix to call `Generate` with `outFile` in that case for now.

Reply:
@haim-kermany, is there a way to have multiple diagrams in a single .drawio file?

Reply:
It can be done